packages/NodeTypeResolver/PHPStan/Scope/NodeVisitor/AssignedToNodeVisitor.php

```diff
@@ -22,6 +23,10 @@ public function enterNode(Node $node): ?Node
             return null;
         }
 
+        if ($node instanceof Coalesce) {
+            $node->var->setAttribute(AttributeKey::IS_ASSIGNED_TO, true);
```

Comment:
I think common `AssignOp` should be used for it, see https://getrector.com/demo/807f2ae3-e2c3-4810-9b52-96030125a160

that still removed.

Reply:
Makes sense :+1:  Could you fix that?

Reply:
I will try 👍

Reply:
https://github.com/rectorphp/rector-src/pull/4506